README.md

```diff
@@ -55,7 +55,7 @@ inside GCB after a job is submitted using `gcbmgr`.
 
 The branch fast-forward tool used between after a new branch as been created.
```

Comment:
maybe also fix this sentence (it has some grammatical issues right now)?

Reply:
@ixdy Good catch. 

How about,

"The branch fast-forward tool is used to pull new patches onto the release branch" ?

I don't particular know too much about branchff. What do you think @hoegaarden? what would be a concise way to explain what branchff does?

Reply:
/sgtm